### PR TITLE
feat(import): Set sorting before import to true as a default

### LINKED ISSUE #56: Sort export lines by category level per default
When importing previously exported categories you have to provide the --sort parameter in order to prevent errors when creating categories their parents are not yet created.

Why not automatically order exported categories by category level?

### PR DESCRIPTION
**Pull request by junajan**
#### Summary
Resolves #56

This PR sets `--sort true` parameter as a default. The sorting feature can be disabled by `--sort false` or `--sort 0`. All other options (eg: `--sort`, `--sort true`, or not provided at all) will resolve as `sort = true`

### CODE REVIEW

File: src/coffee/csv/categorysort.coffee [hunk 1]
```diff
@@ -11,10 +11,13 @@ class CategorySort
       language: @options.language
     }
 
-  getValueByHeader: (row, header, colName) ->
+  getValueByHeader: (row, header, colName, defaultValue) ->
```
[the remark this answers is not in the document]
Comment (junajan) [reply src/coffee/csv/categorysort.coffee]: We run sorting by default now but it does not work when you do not provide parent (externalId, named path or id) column so this default value set parent as empty and all categories will be created as a root cats.

File: README.md [hunk 2]
```diff
@@ -51,8 +51,8 @@ The command line to import or update categories is shown below:
 Usage: bin/category-sync -p <project-key> import -f <CSV file>
 
 Options:
-  -f, --file  CSV file name                          [required]
-  --sort      Sort CSV file before importing
+  -f, --file            CSV file name                          [required]
+  --sort true/false     Sort CSV file before importing (default: true)
```
[the remark this answers is not in the document]
Comment (junajan) [reply README.md]: Please check 44516f7

File: src/coffee/run.coffee [hunk 3]
```diff
@@ -94,6 +94,7 @@ ensureCredentials(argv)
     language: language
     parentBy: parentBy
     continueOnProblems: continueOnProblems
+    sort: true
```
[the remark this answers is not in the document]
Comment (junajan) [reply src/coffee/run.coffee]: Please check 44516f7

File: src/coffee/run.coffee [hunk 4]
```diff
@@ -107,15 +108,17 @@ ensureCredentials(argv)
     .usage 'Usage: $0 -p <project-key> import -f <CSV file>'
     .example '$0 -p my-project-42 import -f categories.csv', 'Import categories from "categories.csv" file into SPHERE project with key "my-project-42".'
 
-    .describe 'sort', 'Sort categories by parentId before importing'
+    .describe 'sort true/false', 'Sort categories by parentId before importing (default: true)'
```
[the remark this answers is not in the document]
Comment (junajan) [reply src/coffee/run.coffee]: 44516f7

File: src/coffee/run.coffee [hunk 5]
```diff
@@ -107,15 +108,17 @@ ensureCredentials(argv)
...
 
     .describe 'f', 'CSV file name'
     .nargs 'f', 1
     .alias 'f', 'file'
     .demand 'f'
     .argv
 
-    options.sort = argv.sort
+    # Sort feature can be disabled by providing --sort false or --sort 0 param
```
[the remark this answers is not in the document]
Comment (junajan) [reply src/coffee/run.coffee]: 44516f7

File: src/coffee/csv/categorysort.coffee (hunk 1 above)
[the remark this answers is not in the document]
Comment (junajan) [reply src/coffee/csv/categorysort.coffee]: comment added here f5686cc

File: src/coffee/run.coffee [hunk 6]
```diff
@@ -103,19 +103,20 @@ ensureCredentials(argv)
...
 
   if command is 'import'
-    yargs.reset()
+    args = yargs.reset()
     .usage 'Usage: $0 -p <project-key> import -f <CSV file>'
     .example '$0 -p my-project-42 import -f categories.csv', 'Import categories from "categories.csv" file into SPHERE project with key "my-project-42".'
-
-    .describe 'sort', 'Sort categories by parentId before importing'
-
+    .describe 'sort', 'Sort categories by parentId before importing.'
```
Comment (wizzy25) [inline src/coffee/run.coffee]: Thanks for the refactor. Just insert line breaks so there is a clearer distinction of where the sort is

Comment (junajan) [reply src/coffee/run.coffee]: Updated 4f5a572

**Pull request merged**